en/2-deeds.md

```diff
@@ -69,47 +72,47 @@ The Deed’s holder or renter can enrich its store with its own perks, connect t
 - Physical office space
 - Etc.
 
-The store functions through the use of Meeds so that Meeds are partially returned to the Deed.
+The store functions through the use of Meeds so that Meeds are partially returned to the community.
 
 ## 2.5. Rewards – Meeds and other tokens
 
 The WoM will tokenize users engagement, by gamifying digital work tools, and attributing points and Meeds in relation to work engagement.
-Services plugged into the engagement and reward engine attribute points to each user, according to their engagement level and in line with each Deed’s engagement programs.
-Indeed, each Deed can fine-tune the amount of points given for each type of activity. Thus, pushing some code in GitHub may be worth 10 points for an IT shop, while a digital agency may attribute only 5 points for GitHub pushes but would offer 10 points for retweets.
+Services plugged into the engagement and reward engine attribute points to each user, according to their engagement level and in line with each Hub's engagement programs.
+Indeed, each community can fine-tune the amount of points given for each type of activity. Thus, pushing some code in GitHub may be worth 10 points for an IT shop, while a digital agency may attribute only 5 points for GitHub pushes but would offer 10 points for retweets.
 
-Periodically, the rewarding engine converts engagement points into Meeds according to budgets and other distribution rules and ultimately rewards the users via their wallets. The Deed is not required to distribute Meeds and can use the engagement system to enable some other form of compensation program. However, distributing Meeds will improve its minting capacity.
+Periodically, the rewarding engine converts engagement points into Meeds according to budgets and other distribution rules and ultimately rewards the users via their wallets. The Hub is not required to distribute Meeds and can use the engagement system to enable some other form of compensation program. However, distributing Meeds will improve its minting capacity.
 
-Moreover, the Deed does not necessarily need to acquire Meeds in order to distribute them and fuel its engagement programs, as it will get Meeds from the “mintium” model, described below.
+Moreover, the community does not necessarily need to acquire Meeds in order to distribute them and fuel its engagement programs, as it will get Meeds from the “mintium” model, described below.
```

Comment:
```suggestion
Moreover, the space does not necessarily need to acquire Meeds in order to distribute them and fuel its engagement programs, as it will get Meeds from the “mintium” model, described below.
```